Test failure on "undo" branch

On the master branch (which tracks upstream), if you run "make", all tests pass. But on the undo branch, tests fail for cmd/dlv, pkg/proc and pkg/terminal. See transcripts below.

### Analysis

* The failure in cmd/dlv is caused by a missed occurrence of "go-delve" in the `projectRoot` function in cmd/dlv/dlv_test.go and is easily fixed: commit de0e64c96fed16139a31df3e17d84f0d027ea3bb.
* The failures in pkg/proc and pkg/terminal are due to running the test under Go 1.13, which no longer autogenerates an init function. To fix this we need Alessandro Arzilli's commit 55eed318fdccdc902ea4975e1d7d174c9a11d55c. Unfortunately this does not cherry-pick cleanly, so I will fix this on another branch.

### Transcript for master

```
$ git checkout master
$ make
Testing default backend
ok  	github.com/go-delve/delve/cmd/dlv	15.830s
?   	github.com/go-delve/delve/cmd/dlv/cmds	[no test files]
ok  	github.com/go-delve/delve/pkg/config	0.002s
?   	github.com/go-delve/delve/pkg/dwarf/dwarfbuilder	[no test files]
ok  	github.com/go-delve/delve/pkg/dwarf/frame	0.005s
?   	github.com/go-delve/delve/pkg/dwarf/godwarf	[no test files]
ok  	github.com/go-delve/delve/pkg/dwarf/line	1.066s
?   	github.com/go-delve/delve/pkg/dwarf/loclist	[no test files]
ok  	github.com/go-delve/delve/pkg/dwarf/op	0.001s
?   	github.com/go-delve/delve/pkg/dwarf/reader	[no test files]
ok  	github.com/go-delve/delve/pkg/dwarf/util	0.001s
ok  	github.com/go-delve/delve/pkg/goversion	0.003s
?   	github.com/go-delve/delve/pkg/logflags	[no test files]
ok  	github.com/go-delve/delve/pkg/proc	78.071s
ok  	github.com/go-delve/delve/pkg/proc/core	0.798s
?   	github.com/go-delve/delve/pkg/proc/core/minidump	[no test files]
ok  	github.com/go-delve/delve/pkg/proc/fbsdutil	0.001s
ok  	github.com/go-delve/delve/pkg/proc/gdbserial	0.820s
ok  	github.com/go-delve/delve/pkg/proc/linutil	0.002s
?   	github.com/go-delve/delve/pkg/proc/native	[no test files]
?   	github.com/go-delve/delve/pkg/proc/test	[no test files]
?   	github.com/go-delve/delve/pkg/proc/winutil	[no test files]
ok  	github.com/go-delve/delve/pkg/terminal	13.791s
?   	github.com/go-delve/delve/pkg/terminal/starbind	[no test files]
?   	github.com/go-delve/delve/pkg/version	[no test files]
?   	github.com/go-delve/delve/service	[no test files]
?   	github.com/go-delve/delve/service/api	[no test files]
ok  	github.com/go-delve/delve/service/debugger	0.002s
?   	github.com/go-delve/delve/service/rpc1	[no test files]
?   	github.com/go-delve/delve/service/rpc2	[no test files]
?   	github.com/go-delve/delve/service/rpccommon	[no test files]
ok  	github.com/go-delve/delve/service/test	59.826s

Testing PIE buildmode, default backend
go test -count 1 -p 1 "-ldflags=-X main.Build=3c266810757cd4abafe46fc6c9203aeeef5e15bf" github.com/go-delve/delve/pkg/proc github.com/go-delve/delve/service/test github.com/go-delve/delve/pkg/terminal -test-buildmode=pie
ok  	github.com/go-delve/delve/pkg/proc	96.559s
ok  	github.com/go-delve/delve/service/test	69.720s
ok  	github.com/go-delve/delve/pkg/terminal	18.998s
go test -count 1 -p 1 "-ldflags=-X main.Build=3c266810757cd4abafe46fc6c9203aeeef5e15bf" github.com/go-delve/delve/pkg/proc/core -test-buildmode=pie
ok  	github.com/go-delve/delve/pkg/proc/core	1.542s
```

### Partial transcript for undo

```
$ git checkout undo
$ make
Testing default backend
--- FAIL: TestGeneratedDoc (0.00s)
    dlv_test.go:225: Documentation/cli/README.md: needs to be regenerated; run scripts/gen-cli-docs.go
FAIL
FAIL	github.com/undoio/delve/cmd/dlv	11.653s
?   	github.com/undoio/delve/cmd/dlv/cmds	[no test files]
ok  	github.com/undoio/delve/pkg/config	0.002s
?   	github.com/undoio/delve/pkg/dwarf/dwarfbuilder	[no test files]
ok  	github.com/undoio/delve/pkg/dwarf/frame	0.001s
?   	github.com/undoio/delve/pkg/dwarf/godwarf	[no test files]
ok  	github.com/undoio/delve/pkg/dwarf/line	1.072s
ok  	github.com/undoio/delve/pkg/dwarf/op	0.001s
?   	github.com/undoio/delve/pkg/dwarf/reader	[no test files]
ok  	github.com/undoio/delve/pkg/dwarf/util	0.001s
ok  	github.com/undoio/delve/pkg/goversion	0.004s
?   	github.com/undoio/delve/pkg/logflags	[no test files]
[ ... output deleted ...]
--- FAIL: TestIssue396 (0.74s)
    proc_test.go:122: failed assertion at proc_test.go:1813: FindFunctionLocation() - Could not find function main.init
[ ... output deleted ...]
FAIL	github.com/undoio/delve/pkg/proc	68.336s
ok  	github.com/undoio/delve/pkg/proc/core	0.816s
?   	github.com/undoio/delve/pkg/proc/core/minidump	[no test files]
ok  	github.com/undoio/delve/pkg/proc/gdbserial	0.814s
?   	github.com/undoio/delve/pkg/proc/linutil	[no test files]
?   	github.com/undoio/delve/pkg/proc/native	[no test files]
?   	github.com/undoio/delve/pkg/proc/test	[no test files]
?   	github.com/undoio/delve/pkg/proc/winutil	[no test files]
--- FAIL: TestDisassembleAutogenerated (0.25s)
    command_test.go:76: Error executing <break main.init>: Location "main.init" not found
FAIL
FAIL	github.com/undoio/delve/pkg/terminal	11.289s
?   	github.com/undoio/delve/pkg/version	[no test files]
?   	github.com/undoio/delve/service	[no test files]
?   	github.com/undoio/delve/service/api	[no test files]
ok  	github.com/undoio/delve/service/debugger	0.002s
?   	github.com/undoio/delve/service/rpc1	[no test files]
?   	github.com/undoio/delve/service/rpc2	[no test files]
?   	github.com/undoio/delve/service/rpccommon	[no test files]
ok  	github.com/undoio/delve/service/test	56.928s
FAIL
exit status 1
Makefile:13: recipe for target 'test' failed
make: *** [test] Error 1
```

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -1,6 +1,6 @@
 version: '{build}'
 os: Windows Server 2012 R2
-clone_folder: c:\gopath\src\github.com\go-delve\delve
+clone_folder: c:\gopath\src\github.com\undoio\delve
 environment:
   GOPATH: C:\gopath
 install:

--- a/cmd/dlv/dlv_test.go
+++ b/cmd/dlv/dlv_test.go
@@ -62,7 +62,7 @@ func projectRoot() string {
 	for _, curpath := range gopaths {
 		// Detects "gopath mode" when GOPATH contains several paths ex. "d:\\dir\\gopath;f:\\dir\\gopath2"
 		if strings.Contains(wd, curpath) {
-			return filepath.Join(curpath, "src", "github.com", "go-delve", "delve")
+			return filepath.Join(curpath, "src", "github.com", "undoio", "delve")
 		}
 	}
 	val, err := exec.Command("go", "list", "-m", "-f", "{{ .Dir }}").Output()

